setup.py

```diff
@@ -52,7 +52,6 @@
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
```

Comment:
Why are there no troves for 3.7 and 3.8?

Reply:
3.8 can't be in there until #111 is merged.  The builds for this are intended to fail - TDD!